osu.Game/Overlays/BeatmapSet/Scores/ScoresContainer.cs

```diff
@@ -13,38 +13,30 @@
 using osu.Game.Beatmaps;
 using osu.Game.Online.API;
 using osu.Game.Online.API.Requests;
+using osu.Game.Screens.Select.Leaderboards;
+using osu.Framework.Bindables;
+using System.Collections.Generic;
+using osu.Game.Rulesets.Mods;
 
 namespace osu.Game.Overlays.BeatmapSet.Scores
 {
     public class ScoresContainer : CompositeDrawable
     {
         private const int spacing = 15;
 
+        public readonly Bindable<BeatmapInfo> Beatmap = new Bindable<BeatmapInfo>();
```

Comment:
indentation?

Reply:
Not sure what do you mean

Reply:
sorry, i believe github page display broke. it showed your new lines at a different indentation level last time i looked.